### PR TITLE
Revised rank_regression.R

### PR DESCRIPTION
**Pull request by Tim-TU**
See commit messages.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -16,6 +16,10 @@
 
 ## New Features
 
+### Parametric Models
+* `rank_regression()`: New arguments `direction` (specifies direction of dependency in the model) and `control` (enables access to control parameters in `optim`).
```
Comment (DavidBarke) [inline NEWS.md:20]: `optim()`

File: R/rank_regression.R [hunk 2]
```diff
@@ -30,6 +30,13 @@
 #' @param distribution Supposed distribution of the random variable.
 #' @param conf_level Confidence level of the interval. If `distribution` is
 #'   `"weibull"` this must be one of `0.9`, `0.95` or `0.99`.
+#' @param direction Direction of the dependence in the regression model.
+#' @param control A list of control parameters (see [optim][stats::optim]).
+#'
+#' `control` only is in use if a three-parametric distribution was specified.
```
Comment (DavidBarke) [inline R/rank_regression.R:36]: is in use only if

File: NEWS.md (hunk 1 above)
Comment (DavidBarke) [reply NEWS.md:20]: better: enables access to argument `control` in `optim()`

File: R/rank_regression_helpers.R [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+  if (!(conf_level %in% c(0.9, 0.95, 0.99))) {
+    stop(
+      "'conf_level' must be 0.90, 0.95 or 0.99",
+      call. = FALSE
+    )
+  }
+
+  # Lookup:
+  conf <- as.character(conf_level)
+  mock_val <- c("0.9" = 1.4, "0.95" = 2, "0.99" = 3.4)[[conf]]
```
Comment (DavidBarke) [inline R/rank_regression_helpers.R:62]: Would it be possible to interpolate in some way for values between 0.9 and 0.99?

**Review by DavidBarke — APPROVED** [review]

File: R/rank_regression_helpers.R (hunk 3 above)
Comment (Tim-TU) [reply R/rank_regression_helpers.R:62]: Unfortunately not.

**Pull request merged**